.github/workflows/docker.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   docker:
+    if: github.repository == 'opensearch-project/opensearch-ci'
```

Comment:
Forks! Getting errors on personal fork when I push to docker/** path. Want to avoid unnecessary build and accidental push in case someone gets access.

Reply:
Image is only updated using this setup. [Docker compose file](https://github.com/opensearch-project/opensearch-ci/blob/main/resources/docker-compose.yml#L4) is responsible for pulling the image into actual installation set up.